### PR TITLE
Add a valid Track.path to dsd.test function

### PR DESCRIPTION
**Pull request by jmuth**
According with our discussion on issue #17, here is my proposal. 

Resolves: #17

### CODE REVIEW

**Comment by faroit** [comment]
I think we don't have to add a dummy track to the repository to keep it clean and tiny

Alternatives could be: 

1. we download the tracks on demand using the [fetch script](https://github.com/faroit/dsdtools/blob/master/data/fetch.sh)
2. create a fake path using the `tmpfile` module

What do you think?

**Comment by jmuth** [comment]
I'm not a huge fan of polluting repo with mock files, but the problem is that if the user_function use Track.path to compute the separation, then it will fail with a fake path.

An idea could be to generate random signal, then fake path, then writing the signal to that location and then providing this path to the user. Is that what you meant ? 

BTW, I cut the test_sample to 3 seconds, so it minimize the impact with only 500k. But I totally agree that it is not very elegant.

**Comment by faroit** [comment]
> An idea could be to generate random signal, then fake path, then writing the signal to that location and then providing this path to the user. Is that what you meant ?

exactly

i often use this chunk for that purpose:

```python
class tempfile:
    """ Context for temporary file.
    Will find a free temporary filename upon entering
    and will try to delete the file on leaving

    """
    def __enter__(self):
        import tempfile as tmp
        self.handle, self.name = tmp.mkstemp()
        return self.name

    def __exit__(self, type, value, traceback):
        try:
            os.close(self.handle)
            os.remove(self.name)
        except OSError as e:
            if e.errno == 2:
                pass
            else:
                raise e
```

**Comment by jmuth** [comment]
Thanks for the snipet and the suggestion of tempfile library. Here is the reviewed version. I put the class tempfile on top of __init__.py file, if you prefer to have it in its own file let me know.

**Comment by jmuth** [comment]
If you have explanations about the CI check tool failure, I will be happy to correct it.

File: dsdtools/__init__.py [hunk 1]
```diff
@@ -16,6 +16,27 @@
 import dsdtools
 
 
+class tempfile:
+    """ Context for temporary file.
+    Will find a free temporary filename upon entering
+    and will try to delete the file on leaving
+
```
Comment (faroit) [inline dsdtools/__init__.py]: this one needs a proper docstring, I guess to pass the test

**Comment by faroit** [comment]
So i apparently it is just a matter of adding a new unit test. I will have a look at this weekend.

**Comment by faroit** [comment]
I will close out this PR. Please add it to [here](https://github.com/sigsep/sigsep-mus-db)

**Pull request closed without merging**